src/util/builtinOperators.ts

```diff
@@ -305,10 +306,10 @@ function generateGFunction(rm: rbac.RoleManager): any {
     if (!rm) {
       value = name1 === name2;
     } else if (args.length === 2) {
-      value = await rm.hasLink(name1, name2);
+      value = (rm as DefaultRoleManager).syncedHasLink(name1, name2);
```

Comment:
You can add a synced `generateGFunction`, so like `generateGFunctionSynced`, then add check the rm whether includes `syncedHasLink`.

Reply:
@Zxilly